source/install-atlas-cli.txt

```diff
@@ -210,77 +210,126 @@ Follow These Steps
 
-                     .. tab:: Debian 9 (Stretch)
-                        :tabid: comm-deb-9
+                           .. tab:: Debian 9 (Stretch)
+                              :tabid: comm-deb-9
 
-                        .. code-block:: sh
+                              .. code-block:: sh
 
-                           echo "deb http://repo.mongodb.org/apt/debian stretch/mongodb-org/5.0 main" | sudo tee /etc/apt/sources.list.d/mongodb-org-5.0.list
+                                 echo "deb http://repo.mongodb.org/apt/debian stretch/mongodb-org/{+mdbVersion+} main" | sudo tee /etc/apt/sources.list.d/mongodb-org-{+mdbVersion+}.list
 
                .. tab:: MongoDB Enterprise Edition
                   :tabid: mdb-ent
 
-                  Create a ``/etc/apt/sources.list.d/mongodb-enterprise.list``
-                  file for MongoDB. Replace ``5.0`` with your
-                  edition of MongoDB.
+                  Select Ubuntu or Debian, then select your version.
         
                   .. tabs::
 
-                     .. tab:: Ubuntu 20.04 (Focal)
-                        :tabid: ent-focal
+                     .. tab:: Ubuntu
+                        :tabid: ubuntu
+
+                        Create a ``/etc/apt/sources.list.d/mongodb-enterprise.list`` file for your version of Ubuntu.
+                        Replace ``{+mdbVersion+}`` with your edition of
+                        MongoDB.
+
+                        .. tabs::
+
+                           .. tab:: Ubuntu 22.04 (Jammy)
+                              :tabid: ent-jammy
+
+                              .. code-block:: sh
+
+                                 echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.com/apt/ubuntu jammy/mongodb-enterprise/{+mdbVersion+} multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-enterprise.list
+
+                           .. tab:: Ubuntu 20.04 (Focal)
+                              :tabid: ent-focal
+
+                              .. code-block:: sh
+
+                                 echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.com/apt/ubuntu focal/mongodb-enterprise/{+mdbVersion+} multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-enterprise.list
+
+                           .. tab:: Ubuntu 18.04 (Bionic)
+                              :tabid: ent-bionic
 
-                        .. code-block:: sh
+                              .. code-block:: sh
 
-                           echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.com/apt/ubuntu focal/mongodb-enterprise/5.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-enterprise.list
+                                 echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.com/apt/ubuntu bionic/mongodb-enterprise/{+mdbVersion+} multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-enterprise.list
```

Comment:
**Question**: Same spacing as above between `ubuntu` and `bionic`.

Reply:
Both should be correct based on existing docs